app/_src/gateway/kong-manager/auth/sessions.md

```diff
@@ -77,6 +95,21 @@ Attribute | Description
 
 {% endif_version %}
 
+{% if_version gte:3.4.x %}
+
+Attribute | Description
+----------|------------
+`cookie_name` | A name for the cookie. <br> For example, `"cookie_name":"kong_cookie"`
```

Comment:
```suggestion
`cookie_name` | A name for the cookie. <br> For example, `"kong_cookie"`
```

Reply:
Most of this section copied from https://github.com/Kong/docs.konghq.com/blob/4ba9fc4983197291bdfed52f97f1f65a334fa523/app/_src/gateway/kong-manager/auth/sessions.md?plain=1#L88-L94. We should keep the same as the above.